gateway/gateway.go

```diff
@@ -929,6 +931,99 @@ func (gateway *HandleT) pendingEventsHandler(w http.ResponseWriter, r *http.Requ
 	_, _ = w.Write([]byte(fmt.Sprintf("{ \"pending_events\": %d }", getIntResponseFromBool(gateway.getWarehousePending(payload)))))
 }
 
+//Warehouse Async Job Handlers
+func (gateway *HandleT) warehouseJobAddHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Why write a handler here and why not just reverse proxy directly to warehouse. Wdyt ? This creates a coupling between gateway and warehouse which we don't want.

Reply:
We can do reverse proxy. Was following pending-events style. Will modify this.